### DISCUSSION TITLE
You can now Vaccinate in The Viral Game!

### DISCUSSION THREAD

**Discussion by customautosys** [quasarframework/quasar#9872] [Show and Tell]
Hey, great news! To support the international push towards vaccination, we have added a Vaccination gameplay feature in The Viral Game! You can watch our video here: https://youtu.be/35GKll6RP9M and download The Viral Game here for Android: https://play.google.com/store/apps/details?id=com.customautosys.viralgame or visit https://customautosys.com/products/covidgame for The COVID-19 Game for desktop and web (web version works on iOS)! Unfortunately, this is still using the old version of Quasar as the new version of Quasar has just been released so I haven't had the time to port it yet.

**Comment by customautosys** [comment]
I am humbled to have attained the award of Most Pioneering Legal & Fintech Development Company - Southeast Asia from Corporate Vision!
https://www.corporatevision-news.com/winners/custom-automated-systems-pte-ltd
![Screenshot_20210630_203052_org.mozilla.firefox.png](https://user-images.githubusercontent.com/14951259/124006321-ae14e600-da0c-11eb-9a23-9bff5c37e392.png)

**Reply by ontwikkelfabriek** [reply]
@siauderman 

awesome, going to install it on android! Btw what is that 'debugger' looping trick you do when you view the source in the web app version?

**Reply by customautosys** [reply]
I put this in extendWebpack(cfg) in quasar.conf.js:
````
if(!ctx.dev&&!ctx.debug&&!ctx.mode.bex){
	cfg.plugins.push(new JavaScriptObfuscator({
		debugProtection:true,
		debugProtectionInterval:!ctx.mode.cordova,
		rotateStringArray:true,
		renameGlobals:!ctx.mode.cordova,
		stringArrayEncoding:'rc4',
		stringArrayThreshold:1
	}));
}
````
On cordova, renameGlobals stops the game from working and debugProtectionInterval lags the app massively.

This is Quasar 1, by the way.
